app/src/main/java/com/infomaniak/mail/data/models/thread/Thread.kt

```diff
@@ -118,4 +116,8 @@ class Thread : RealmObject {
         ATTACHMENTS(R.string.searchFilterAttachment),
         FOLDER(R.string.searchFilterFolder),
     }
+
+    companion object {
```

Comment:
```suggestion
    private companion object {
```

Reply:
This unfortunately makes the app crash because `formatDate()` can't access it anymore